.config-schema.json

```
@@ -73,6 +73,17 @@
             "default": [50, 200],
             "x-env-variable": "OPENFGA_REQUEST_DURATION_DATASTORE_QUERY_COUNT_BUCKETS"
         },
+        "requestDurationDispatchCountBuckets": {
+            "description": "Datastore dispatch count buckets used to label the histogram metric for measuring request duration.",
```

```suggestion
            "description": "Dispatch count buckets used to label the histogram metric for measuring request duration.",
```

Dispatch is not related to datastore